File: JAVA/BasicForProgramming/Armstrong.java
import java.util.*;

public class Armstrong {
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);

		int n = in.nextInt();
		int temp = n, sumn = 0;

		while (temp > 0) {
			int dgt  = temp % 10;
			sumn += Math.pow(dgt, 3);
			temp /= 10;
		}
		System.out.println((n == sumn));
	}
}

File: CPP/BasicForProgramming/Armstrong.cpp
#include <iostream>

using namespace std;

int main() {
	int n;  cin >> n;
	int temp = n, sumn = 0;

	while (temp > 0) {
		int dgt = temp % 10;
		sumn += dgt * dgt * dgt;
		temp /= 10;
	}
	if (sumn = n) { cout << "Armstrong" << endl; }
	else { cout << "Not Armstrong" << endl; }
	return 0;
}